app/models/organization.rb

```diff
@@ -14,6 +14,27 @@ class Organization < ApplicationRecord
 
   auto_strip_attributes :name, squish: true, upcase_first: true
 
+  # ----------------
+  # Region callbacks
+  after_create :create_region
+  before_update :update_region
+
+  def create_region
+    return if region&.persisted?
```

Comment:
This will handle the case where Regions are driving things in the future, so we don't accidentally create unnecessary regions where one is already created.

Reply:
Make sense!